release.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+function main() {
+  parse_args $@
+  
+  if [ $h ]; then 
```

Comment:
Either test for variable being non-empty using -n / -z or define the variable values to be `true` and `false` which are guaranteed to be commands returning zero and nonzero respectively and call them directly as in `if "$h"; then`. Either way define default values of the variables (or `unset` them) in parse_args() so they can't leak in from somewhere.